### ISSUE TITLE
How to build Setup?

### ISSUE THREAD

**Issue by artfabrique** [ShareX/ShareX#3725]
I've made a fork with couple changes to Custom Uploader - it now supports PUT requests with files/images. I want to setup ShareX across all our organization (~70 computers). I've built ShareX.Setup with no error but when I run it I get this:
```
PS D:\WECANDO\Dropbox\Revenga\ShareX\ShareX.Setup\bin\Release> .\ShareX.Setup.exe
ShareX setup started.
Setup job: Stable
Cleaning output directory: ..\..\..\Output
InnoSetup compiler is missing: C:\Program Files (x86)\Inno Setup 5\ISCC.exe
InnoSetup compiler is missing: C:\Program Files (x86)\Inno Setup 5\ISCC.exe
Creating folder: ..\..\..\Output\ShareX-portable
InnoSetup compiler is missing: C:\Program Files (x86)\Inno Setup 5\ISCC.exe

Unhandled Exception: System.IO.FileNotFoundException: Could not find file '..\..\..\Output\Recorder-devices-setup.exe'.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.File.InternalCopy(String sourceFileName, String destFileName, Boolean overwrite, Boolean checkHost)
   at ShareX.Setup.SetupHelpers.CopyFiles(IEnumerable`1 files, String toFolder)
   at ShareX.Setup.Program.CreateFolder(String source, String destination, SetupJobs job)
   at ShareX.Setup.Program.Main(String[] args)
```

**Comment by artfabrique** [comment]
Ok, I've figured out that I need 3rd party software - Inno Setup (http://www.jrsoftware.org/isdl.php)
But it produces Portable version in the Output folder instead of regular Sharex setup exe.

**Comment by stuntguy3000** [comment]
Whilst I can't answer your question, maybe take a look at the way AppVeyor does it?

Also, I suspect you will run into issues when ShareX releases updates. Maybe look into a different method of doing this.

**Comment by Jaex** [comment]
Closing this because issue resolved in #3717
